### DISCUSSION TITLE
Recommendation for project structure

### DISCUSSION THREAD

**Discussion by krokoko** [projen/projen#2958] [Q&A] [answered]
Hi team,

First, thanks again for Projen, awesome tool !

One question here related to project structure: in multiple projects we have projenrc files which start to be really long. Is there any official recommendation on how we can split the rc file into several sub files ? 
The current solution is to have a projenrc folder, which contains multiple Ts files, and they are imported by the main projenrc file.
I don't recall seeing anything in the documentation about this. 

Thank you !

**Comment by mrgrain** [comment, marked as answer]
Yeah good call-out re missing docs. Using a projenrc folder is definitely the way to go. The next step up is to publish custom project types as packages (which is also not documented).

**Comment by krokoko** [comment]
Awesome thanks @mrgrain !

**Reply by mrgrain** [reply]
I can point you to my package for project types as inspiration:
https://github.com/mrgrain/projen-projects
